spec/validation.spec.js

```diff
@@ -216,7 +216,7 @@ describe("Request validation", () => {
 
   it("should validate urls with special characters", () => {
     const specialRequestUrl = requestUrl + "&Body=It's+amazing";
-    const signature = "dsq4Ehbj6cs+KdTkpF5sSSplOWw=";
+    const signature = "TfZzewPq8wqrGlMfyAud8+/IvJ0=";
```

Comment:
this changed because `URL` encodes single quotes, whereas `url-parse` doesn't

that means the signature is different, but should be ok since `url-parse` was wrong/off-spec basically

probably is worth you double checking i got the right signature though

Reply:
Ahh got it, I checked. The encoding differs in URL than url-parse. This is correct I think, url-parse wasn't encoding quotes and hashtags. The signature is correct, it passes the tests.